tedana/utils/utils.py

```diff
@@ -169,6 +169,9 @@ def make_adaptive_mask(data, minimum=True, getsum=False):
     data : (S x E x T) array_like
         Multi-echo data array, where `S` is samples, `E` is echos, and `T` is
         time
+    mask : :obj:`str`, optional
+        Binary mask for voxels to consider in TE Dependent ANAlysis. Default is
+        to generate mask from data with good signal across echoes
```

Comment:
This can be a string (filename), a numpy array, or an image, right?

Reply:
Right now I'm only accepting a file name, since it's user-specified. Can you explain how it could be an image or numpy array ?